logstash-input-syslog.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |s|
   # Gem dependencies
   s.add_runtime_dependency 'logstash', '>= 1.4.0', '< 2.0.0'
 
+  s.add_runtime_dependency 'atomic'
```

Comment:
NOT https://github.com/ruby-concurrency/concurrent-ruby/blob/master/lib/concurrent/atomic.rb

I guess I will update.